aicsimageio/readers/czi_reader.py

```diff
@@ -213,3 +213,7 @@ def _is_multiscene(self):
             return False
         img_shape = self.czi.filtered_subblock_directory[0].shape
         return img_shape[index] != 1
+
+    def get_channel_names(self, scene=0):
+        chelem = self.metadata.findall("./Metadata/Information/Image/Dimensions/Channels/Channel")
```

Comment:
I think this should be just `/Metadata/...` not `./Metadata/...` We want to always route from the root object not from current (in the case that the metadata was mutated)

Reply:
I tried changing this and got an error: ` SyntaxError: cannot use absolute path on element`